src/app/js/formats/streamgraph/utils.js

```diff
@@ -0,0 +1,204 @@
+
+import differenceBy from 'lodash/differenceBy';
+import * as d3 from 'd3';
+
+// ===========================================================================================
+// getCssToString get the css js object and return it as string
+// ===========================================================================================
+
+export function getCssToString(cssToChange, styles) {
+    const cssString = Object.keys(styles.legend).map(key => {
```

Comment:
You don't have to translate styles into CSS: aphrodite can do that for you.
Just apply the correct class names.